boulderlog/Program.cs

```diff
@@ -147,6 +147,19 @@ public static void Main(string[] args)
                 name: "default",
                 pattern: "{controller=Home}/{action=Index}/{id?}");//.RequireRateLimiting(rateLimitOptions.Policy);
             app.MapRazorPages();//.RequireRateLimiting(rateLimitOptions.Policy);
+            app.Use(async (context, next) =>
+           {
+              await next();
+
+            if (context.Response.StatusCode == 404)
+            {
+                 context.Response.Redirect("/"); //Redirecting if wrong url or wrong route is added
+            }
+            else if (!context.User.Identity.IsAuthenticated)
+            {
+                 context.Response.Redirect("Identity/Account/Login");
+            }
```

Comment:
```suggestion
```

Hi @apchandu96, I did some testing, and it appears the auth routes already redirect the login page. Thank you again for this contribution, I'll approve the pr once you update this change to take out the login redirect.